tests/automated_unit_tests/comms/test_factory_payload_sender.py

```diff
@@ -0,0 +1,194 @@
+import os
+import sys
+import mock
+
+sys.path.append('./src')
+
+from asmcnc.production.database.factory_payload_sender import get_csv, send_csv_to_ftp, CSV_PATH
+
```

Comment:
Add comments containing details of how to run these tests at the top of the file